p2p_wallet/UIKit/Views/WLFloatingPanelView.swift

```diff
@@ -17,7 +17,7 @@ class WLFloatingPanelView: BERoundedCornerShadowView {
     
     override func commonInit() {
         super.commonInit()
-        layer.cornerRadius = 12
+        layer.cornerRadius = 8
```

Comment:
There are many places we use cornerRadius = 12, so don't change this value just for layouting 1 view that you are working on.
To change the BalancesOverView's cornerRadius, put this line in `Home.RootView.swift`, line 58
```swift
view.layer.cornerRadius = 8
```